src/System.Linq.Parallel/tests/Combinatorial/CancellationParallelQueryCombinationTests.cs

```diff
@@ -33,6 +33,11 @@ public static void Aggregate_AggregateException_Wraps_OperationCanceledException
             Functions.AssertAggregateAlternateCanceled((token, canceler) => Cancel(token, canceler, source, operation).Aggregate(0, (i, j) => j, i => i));
             Functions.AssertAggregateAlternateCanceled((token, canceler) => Cancel(token, canceler, source, operation).Aggregate(0, (i, j) => j, (i, j) => i, i => i));
             Functions.AssertAggregateAlternateCanceled((token, canceler) => Cancel(token, canceler, source, operation).Aggregate(() => 0, (i, j) => j, (i, j) => i, i => i));
+            Functions.AssertAggregateNotCanceled((token, canceler) => Cancel(token, canceler, source, operation).Aggregate((i, j) => j));
```

Comment:
Do we really need these NotCanceled tests for all of these different operators?  Is it exercising code paths that the AlternateCanceled didn't cover?  I'm not convinced that all of these extra tests add much more value than the original test that was there.